numpy/random/_generator.pyx

```diff
@@ -2083,7 +2083,7 @@ cdef class Generator:
         -----
         The probability density for the Pareto II distribution is
 
-        .. math:: p(x) = \\frac{a}{{x+1}^{a+1}} , x \ge 0
+        .. math:: p(x) = \\frac{a}{{(x+1)}^{a+1}} , x \ge 0
```

Comment:
And while we're at it; the $x \ge 0$ could use a bit of breathing room;

```suggestion
        .. math:: p(x) = \\frac{a}{{(x+1)}^{a+1}} ,\\ x \ge 0
```

before

$$
\frac{a}{{(x+1)}^{a+1}} , x \ge 0
$$

after


$$
\frac{a}{{(x+1)}^{a+1}} ,\ x \ge 0
$$

I'm not sure if the escaped `\\` is needed, but it probably wouldn't hurt